src/bugbug/eval/requirements.nix

```diff
@@ -0,0 +1,2132 @@
+      doCheck = commonDoCheck;
+      checkPhase = "";
+      installCheckPhase = "";
+      buildInputs = commonBuildInputs;
+      propagatedBuildInputs = [ ];
+      meta = with pkgs.stdenv.lib; {
+        homepage = "https://github.com/PyCQA/pyflakes";
+        license = licenses.mit;
+        description = "passive checker of Python programs";
+      };
+    };
+
+    "pytest" = python.mkDerivation {
+      name = "pytest-4.0.1";
+      src = pkgs.fetchurl { url = "https://files.pythonhosted.org/packages/24/f1/0e378fa418d9ac15d2d28296be916a55e351a6ffeb74105fe333c15ea58a/pytest-4.0.1.tar.gz"; sha256 = "1d131cc532be0023ef8ae265e2a779938d0619bb6c2510f52987ffcba7fa1ee4"; };
+      doCheck = commonDoCheck;
+      checkPhase = "";
+      installCheckPhase = "";
+      buildInputs = commonBuildInputs;
+      propagatedBuildInputs = [
+      self."atomicwrites"
+      self."attrs"
+      self."more-itertools"
+      self."pluggy"
+      self."py"
+      self."requests"
+      self."six"
+    ];
+      meta = with pkgs.stdenv.lib; {
+        homepage = "https://docs.pytest.org/en/latest/";
+        license = licenses.mit;
+        description = "pytest: simple powerful testing with Python";
+      };
+    };
+
+    "pytest-cov" = python.mkDerivation {
+      name = "pytest-cov-2.6.0";
+      src = pkgs.fetchurl { url = "https://files.pythonhosted.org/packages/d9/e2/58f90a316fbd94dd50bf5c826a23f3f5d079fb3cc448c1e9f0e3c33a3d2a/pytest-cov-2.6.0.tar.gz"; sha256 = "e360f048b7dae3f2f2a9a4d067b2dd6b6a015d384d1577c994a43f3f7cbad762"; };
+      doCheck = commonDoCheck;
+      checkPhase = "";
+      installCheckPhase = "";
+      buildInputs = commonBuildInputs;
+      propagatedBuildInputs = [
+      self."coverage"
+      self."pytest"
+    ];
+      meta = with pkgs.stdenv.lib; {
+        homepage = "https://github.com/pytest-dev/pytest-cov";
+        license = licenses.bsdOriginal;
+        description = "Pytest plugin for measuring coverage.";
+      };
+    };
+
+    "python-dateutil" = python.mkDerivation {
+      name = "python-dateutil-2.6.1";
+      src = pkgs.fetchurl { url = "https://files.pythonhosted.org/packages/54/bb/f1db86504f7a49e1d9b9301531181b00a1c7325dc85a29160ee3eaa73a54/python-dateutil-2.6.1.tar.gz"; sha256 = "891c38b2a02f5bb1be3e4793866c8df49c7d19baabf9c1bad62547e0b4866aca"; };
+      doCheck = commonDoCheck;
+      checkPhase = "";
+      installCheckPhase = "";
+      buildInputs = commonBuildInputs;
+      propagatedBuildInputs = [
+      self."six"
+    ];
+      meta = with pkgs.stdenv.lib; {
+        homepage = "https://dateutil.readthedocs.io";
+        license = licenses.bsdOriginal;
+        description = "Extensions to the standard Python datetime module";
+      };
+    };
+
+    "python-hglib" = python.mkDerivation {
+      name = "python-hglib-2.6.1";
+      src = pkgs.fetc
```

Comment:
I've had to add this manually, as if I had `https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.0.0/en_core_web_sm-2.0.0.tar.gz#egg=en_core_web_sm` in requirements.txt, pypi2nix would fail trying to request some data from pypi.org.

Reply:
CC @garbas